admin/sql/create_tables.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_row_id_key UNIQUE (musicbrainz_row_id);
+ALTER TABLE "user" ADD COLUMN alt_id VARCHAR;
```

Comment:
@vansika, the bug seems to be here, you're declaring the `alt_id` column as varchar, but then comparing it with integers later.

Also, this column should be inside the `CREATE TABLE` statement instead of being a seperate `ALTER` statement.

Reply:
Please note that the automated tests on travis do a fresh build of the code you push, while your local environment may be different because of experimentation etc.

Reply:
@paramsingh, get_id( ) passes alt_id to load_user( ) which is a string, but the redirects in tests are passing user_id to load_user( ), which is the cause of error because the check function (filter_by_alt_id( )) inside load_user( ) can return user by checking against alt_id (string) and not user_id (integer). 
I will try to debug. Thank you